app/views/complaints/report.html.erb

```diff
@@ -0,0 +1,127 @@
+<%= link_to safety_center_path do %>
+  &laquo; Back to Safety Center
+<% end %>
+
+<h1>Make a report or complaint</h1>
+<p>
+  Thank you for taking the time to make a report. If you've seen harmful, abusive, or illegal content on our
+  communities, you can report this to us here. You can also use this page if you've received a message saying we've
+  classified your content as harmful, abusive, or illegal and you wish to contest it.
+</p>
+
+<div class="notice is-warning has-color-yellow-900 flex-row-always ai-c has-font-size-caption h-m-b-4">
+  <i class="fas fa-exclamation-triangle"></i>
+  <p>
+    Please don't use this tool to complain about moderator actions, including deletion of posts. Raise these in the

```

Comment:
Nit: let's say "moderation actions", not "moderator actions", to include stuff that curators and other non-mods can do.